api/handlers/github.ts

```diff
@@ -87,6 +87,7 @@ export const markBuildArtifactsForDeletion = async (event: APIGatewayEvent) => {
 
   const { repository, pull_request: pullRequest } = payload;
   const branch = pullRequest.head.ref;
+  const user = pullRequest.user.login;
```

Comment:
question for myself - where is this payload constructed?

from [this block](https://github.com/octokit/webhooks/blob/v7.1.0/payload-types/schema.d.ts#L5596), it seems like it should be `sender` attribute? CMIIW

Reply:
I believe it should be the `PullRequestClosedEvent` object, but yes, `sender` is present there too. The only thing I'm unsure/concerned about is if `sender` corresponds with the user who's actually closing or merging the PR, and if the person closing the PR is not the same person who authored the PR, then we could potentially be checking for the wrong user

Reply:
:+1: https://github.com/octokit/webhooks/blob/v7.1.0/payload-types/schema.d.ts#L2819